start/sw/web-app/laravel5.md

```diff
@@ -7,91 +7,86 @@ order: 3
 ## Laravel 5
 > The PHP Framework For Web Artisans
 
-Laravel is a high-level PHP Web framework 
+Laravel is a high-level PHP Web framework. We have to make sure all of these requirements are satisfied.
```

Comment:
I would reword the second sentence as "To use (or install) Laravel the following needs to be satisfied:"

The sentence as it is now is somehow out of context.